lib/iris/coords.py

```diff
@@ -1464,12 +1464,12 @@ def __common_cmp__(self, other, operator_method):
                 # - Simple matching
                 me = self.point
             else:
-                if hasattr(other, "timetuple"):
-                    raise TypeError(
-                        "Cannot determine whether a point lies "
-                        "within a bounded region for "
-                        "datetime-like objects."
-                    )
+                # if hasattr(other, "timetuple"):
+                #     raise TypeError(
+                #         "Cannot determine whether a point lies "
+                #         "within a bounded region for "
+                #         "datetime-like objects."
+                #     )
```

Comment:
Please could this be removed?

We always remove commented code, as it often gets forgotten. If we need to get something back we rely on Git.

Reply:
Sorry, left that in from testing, you're entirely right.